From 3ecce781aeb3a71f31648589257c65a515382d7d Mon Sep 17 00:00:00 2001
From: Samreen Nilofer <samreen.nilofer@intel.com>
Date: Tue, 3 Mar 2015 15:25:33 +0530
Subject: [PATCH] ASoC: Intel: Update the dpcm_playback variable for Compress
 Dai

Issue: GMINL-6632

dpcm_playback needs to be updated to indicate the compress stream
supports only playback stream.

Change-Id: I8391469dae8530b0d30002f433fcf98006e1aa3d
Signed-off-by: Samreen Nilofer <samreen.nilofer@intel.com>
---
 sound/soc/intel/board/cht_bl_dpcm_rt5672.c | 1 +
 sound/soc/intel/board/cht_cr_aic3100.c     | 1 +
 2 files changed, 2 insertions(+)

diff --git a/sound/soc/intel/board/cht_bl_dpcm_rt5672.c b/sound/soc/intel/board/cht_bl_dpcm_rt5672.c
index 4fc982e..df5a6cb 100644
--- a/sound/soc/intel/board/cht_bl_dpcm_rt5672.c
+++ b/sound/soc/intel/board/cht_bl_dpcm_rt5672.c
@@ -798,6 +798,7 @@ static struct snd_soc_dai_link cht_dailink[] = {
 		.ignore_suspend = 1,
 		.dynamic = 1,
 		.compr_ops = &cht_compr_ops,
+		.dpcm_playback = 1,
 	},
 	[CHT_DPCM_VOIP] = {
 		.name = "Cherrytrail VOIP Port",
diff --git a/sound/soc/intel/board/cht_cr_aic3100.c b/sound/soc/intel/board/cht_cr_aic3100.c
index 2a0d52d..3efbda2 100644
--- a/sound/soc/intel/board/cht_cr_aic3100.c
+++ b/sound/soc/intel/board/cht_cr_aic3100.c
@@ -809,6 +809,7 @@ static struct snd_soc_dai_link cht_dailink[] = {
 		.ignore_suspend = 1,
 		.dynamic = 1,
 		.compr_ops = &cht_compr_ops,
+		.dpcm_playback = 1,
 	},
 	[CHT_DPCM_VOIP] = {
 		.name = "Cherrytrail VOIP Port",
-- 
1.9.1

